server/events/instrumented_project_command_runner.go

```diff
@@ -51,6 +57,10 @@ func RunAndEmitStats(commandName string, ctx command.ProjectContext, execute fun
 		return result
 	}
 
+	// Log successful policy check results
```

Comment:
not a blocker but we could move this into the `PolicyCheck` function and just look at result.PolicyCheckSuccess and log there to avoid checking the command name.

Reply:
Since the code already uses the instrumentation wrapper design, I wanted to unify all stats and logs at this level without having to inject the log into the regular command runner.